src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9407 @@
+        " continue their current action for at least ";
+    m_actionTemplates[ScriptAction::TEAM_SPIN_FOR_FRAMECOUNT].m_uiStrings[2] = " frames.";
+
+    m_actionTemplates[ScriptAction::CAMERA_FADE_MULTIPLY].m_internalName = "CAMERA_FADE_MULTIPLY";
+    m_actionTemplates[ScriptAction::CAMERA_FADE_MULTIPLY].m_uiName =
+        "Camera_/Fade Effects/Fade using a multiply blend to black.";
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_internalName = "CAMERA_ENABLE_SLAVE_MODE";
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_uiName =
+        "Camera_/Enable 3DSMax Camera Animation Playback mode.";
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_numParameters = 2;
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_parameters[0] = Parameter::TEXT_STRING;
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_parameters[1] = Parameter::TEXT_STRING;
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_numUiStrings = 2;
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_uiStrings[0] =
+        "Enable 3DSMax Camera playback of animation with thing name ";
+    m_actionTemplates[ScriptAction::CAMERA_ENABLE_SLAVE_MODE].m_uiStrings[1] = " containing bone name ";
+
+    m_actionTemplates[ScriptAction::CAMERA_DISABLE_SLAVE_MODE].m_internalName = "CAMERA_DISABLE_SLAVE_MODE";
+    m_actionTemplates[ScriptAction::CAMERA_DISABLE_SLAVE_MODE].m_uiName =
+        "Camera_/Disable 3DSMax Camera Animation Playback mode.";
+    m_actionTemplates[ScriptAction::CAMERA_DISABLE_SLAVE_MODE].m_numParameters = 0;
+    m_actionTemplates[ScriptAction::CAMERA_DISABLE_SLAVE_MODE].m_numUiStrings = 1;
+    m_actionTemplates[ScriptAction::CAMERA_DISABLE_SLAVE_MODE].m_uiStrings[0] = "Disable camera playback mode.";
+
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_internalName = "CAMERA_ADD_SHAKER_AT";
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_uiName = "Camera_/Add Camera Shaker Effect at.";
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_numParameters = 4;
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_parameters[0] = Parameter::WAYPOINT;
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_parameters[1] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_parameters[2] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_parameters[3] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_numUiStrings = 4;
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_uiStrings[0] = "Add Camera Shaker Effect at waypoint ";
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_uiStrings[1] = " with Amplitude ";
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_uiStrings[2] = " Duration (seconds) ";
+    m_actionTemplates[ScriptAction::CAMERA_ADD_SHAKER_AT].m_uiStrings[3] = " Radius.";
```

Comment:
This sentence does not look right. It does not accommodate the number of arguments.

Reply:
This is correct.